src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -88,13 +88,17 @@
             <label for="studentNationality">
                 Nationality
             </label>
-            <input id="studentNationality"
-                   name="<%= Const.ParamsNames.STUDENT_NATIONALITY %>"
-                   class="form-control"
-                   type="text"
-                   data-actual-value="${profile.nationality}"
-                   value="${profile.nationality}"
-                   placeholder="Nationality">
+            <select id="studentNationality"
+                    name="<%=Const.ParamsNames.STUDENT_NATIONALITY%>"
+                    class="form-control" style="width: 300px">
+                <c:forEach items="${profile.nationalitySelectField}"
+                           var="option">
+                    <option ${option.attributesToString}>
+                        ${option.content}
+                    </option>
+                </c:forEach>
+            </select>
+            ${profile.legacyNationality}
```

Comment:
Can you produce two screenshots of this portion of the page, displaying how both invalid (legacy) and valid data are handled? To put legacy data into the server, simply add a student profile using the application run from the current `master` branch (or any branch without this PR's changeset).
I will allow those screenshots to serve as the proof that both legacy and current data are properly handled in place of automated tests, at least for now.